bot/exts/info/help.py

```diff
@@ -173,8 +173,15 @@ async def command_formatting(self, command: Command) -> Embed:
         if aliases:
             command_details += f"**Can also use:** {aliases}\n\n"
 
-        # check if the user is allowed to run this command
-        if not await command.can_run(self.context):
+        # when command is disabled, show message about it,
+        # when other CommandError instance is raised, log warning about it
+        # otherwise check if the user is allowed to run this command
+        try:
+            if not await command.can_run(self.context):
+                command_details += "***You cannot run this command.***\n\n"
+        except DisabledCommand:
+            command_details += "***This command is disabled.***\n\n"
+        except CommandError:
             command_details += "***You cannot run this command.***\n\n"
```

Comment:
Should use a variable to ensure the message is the same between it returning `False` and raising a `CommandError`. Could even go further and have a Markdown + newline template within which to format the message.